kasparovd/controllers/block.go

```diff
@@ -55,15 +55,22 @@ func GetBlockByHashHandler(blockHash string) (interface{}, error) {
 
 // GetBlocksHandler searches for all blocks
 func GetBlocksHandler(order string, skip uint64, limit uint64) (interface{}, error) {
-	if limit < 1 || limit > maxGetBlocksLimit {
```

Comment:
Please reconsider this implicit behavior and if you insist on keeping it, please document it.